docs/content-server-api/pointerChanges/get.md

```diff
@@ -0,0 +1,76 @@
+# Get Pointer Changes
+
+List all deployment changes from the same pointer
+
+**URL** : `/pointerChanges`
+
+<!-- **URL Parameters** : `pk=[integer]` where `pk` is the ID of the Account on the server. -->
+
+**Method** : `GET`
+
+**Auth required** : NO
+
+**Permissions required** : NO
+
+**Query Parameters** :
+
+
+
+*DEPRECATED: PLEASE USE from*
```

Comment:
Maybe it makes sense to avoid adding these? I mean, adding documentation about deprecated parameters sounds weird to me. The less people know about these, the easier it will be to remove them